packages/server/src/api/controllers/application.ts

```diff
@@ -278,7 +272,10 @@ export async function fetchAppPackage(
     screens = await accessController.checkScreensAccess(screens, userRoleId)
   }
 
-  if (await features.flags.isEnabled(FeatureFlag.WORKSPACE_APPS)) {
+  if (
+    (await features.flags.isEnabled(FeatureFlag.WORKSPACE_APPS)) &&
+    !isBuilder
```

Comment:
How come this is required now?

Reply:
This endpoint is used in both client and builder for "different" use cases. On the builder we load all the navigations and screens, so not having this check will just load the current app screens, not being able to jump from one app to the other